purchase_stock_picking_return_invoicing/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services
+#           <contact@eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Stock Picking Return Invoicing",
+    "summary": "Add an option to refund returned pickings",
+    "version": "10.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://github.com/OCA/account-invoicing",
+    "author": "Eficent, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "purchase",
+        "purchase_open_qty",
```

Comment:
This shouldn't depend on purchase_open_qty. Why this dependency?

Reply:
I have removed temporarily the dependency for using this PR.

Reply:
Without that dependency, the `qty_to_invoice` won't be visible in the view. So, I would like to have that dependency.

Reply:
But that's not a good modularity, because the module makes other thing and includes that feature, but forcing to have that feature for only a visual field is not the correct way. You can do it for example with fields_view_get, or at the end, merge both modules together if the functionality is very tied to this.

Reply:
It's not only a visual field, both modules define the field <code>qty_to_invoice</code> with a compute.. We discussed that before in https://github.com/OCA/account-invoicing/pull/285. In v9, both modules were already merged and thus we proceeded with the glue module solution (https://github.com/OCA/account-invoicing/pull/297). But now, this module is not merged yet, so we thought that it's better to add the dependency that migrating and maintaining a glue module.